src/diffusers/schedulers/scheduling_pndm_flax.py

```diff
@@ -148,7 +148,8 @@ def __init__(
         # mainly at formula (9), (12), (13) and the Algorithm 2.
         self.pndm_order = 4
 
-        self.state = PNDMSchedulerState.create(num_train_timesteps=num_train_timesteps)
+    def create_state(self):
```

Comment:
cc @pcuenca @kashif what do you think about the design?

Reply:
having a look thanks! I am also adding the scheduler tests too so this will be helpful I believe